src/main/java/core/basesyntax/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.operation;
+
+public interface OperationHandler {
+    int getQuantity(int defaultQuantity, int fruitQuantity);
```

Comment:
The method signature 'getQuantity(int defaultQuantity, int fruitQuantity)' is a bit confusing. It's not clear what 'defaultQuantity' and 'fruitQuantity' represent. Consider renaming these parameters to better reflect their purpose. Additionally, the method name 'getQuantity' is vague; it should describe the operation it's performing more accurately.